src/Illuminate/Foundation/helpers.php

```diff
@@ -443,7 +443,7 @@ function encrypt($value)
     function has_errors($key = null, $bag = null)
     {
         $errors = session('errors');
-        if (if (is_null($errors) || ! $errors->any())) {
+        if (is_null($errors) || ! $errors->any()) {
```

Comment:
Also it always checks the default bag, even if a bag has been specified.

Example of fix:
`is_null($errors) || ! $errors->getBag($bag ?: 'default')->any()`